### ISSUE TITLE
Not working with react-native 0.4.3

### ISSUE THREAD

**Issue by PhilippKrone** [corymsmith/react-native-icons#32]
Hi,

the library is not working with react-native 0.4.3 - various error messages occur.

Regards
Philipp

**Comment by liubko** [comment]
Error: 

```
"Error: Requiring unknown module "ReactIOSViewAttributes".
If you are sure the module is there, try restarting the packager.
```

**Comment by corymsmith** [comment]
There's a PR that fixes this, I'll be merging it in shortly and pushing to NPM. I'll let you know when it's done

**Comment by liubko** [comment]
Oh, I see they renamed it to `ReactNativeViewAttributes`. 
Cool, thanks!

**Comment by corymsmith** [comment]
Fixed and published as 0.0.7
